.github/workflows/release-embedding-sdk.yml

```diff
@@ -249,7 +248,12 @@ jobs:
         run: |
           cd sdk
           echo //registry.npmjs.org/:_authToken=${{ secrets.NPM_RELEASE_TOKEN }} > .npmrc
-          npm publish
+          npm publish --tag ${{fromJson('{"master": "canary", "release-x.51.x": "50-stable"}')[inputs.branch]}}
```

Comment:
Please see the [test results in the PR description](https://github.com/metabase/metabase/pull/48982#issue-2604928879) to verify that this logic works.

This maps the branch name to their respective desired npm dist-tags based on our [spec in the product doc.](https://www.notion.so/metabase/Improve-npm-release-process-11e69354c901808687bcfb8368fd0327?pvs=4#11f69354c9018053ac82de5f7fb8644d)

- `release-x.51.x` -> `50-stable`, `latest`
- `master` -> `@canary`